.github/renovate.json

```diff
@@ -1,6 +1,8 @@
 {
   "$schema": "https://raw.githubusercontent.com/renovatebot/renovate/master/renovate-schema.json",
   "extends": ["config:base"],
+  "automerge": true,
+  "automergeType": "branch",
```

Comment:
I re-enabled the automerge as a follow up to Rhys Arkins reply on https://github.com/renovatebot/config-help/issues/381#issuecomment-543155442 
I think the problem was that the configuration wasn't up-to-date on master.
We'll see how things work when this gets merged into master.

Reply:
Nice, let's merge it right now